pom.xml

```
@@ -11,41 +11,76 @@
   <name>cwa-verification-server</name>
   <description>CWA verification server project.</description>
 
```

Add URLs to be picked up by Sonar:

```suggestion

   <url>https://www.coronawarn.app/</url>
   <ciManagement>
     <url>https://github.com/corona-warn-app/cwa-verification-server/actions?query=workflow%3Aci</url>
   </ciManagement>
   <issueManagement>
     <url>https://github.com/corona-warn-app/cwa-verification-server/issues</url>
   </issueManagement>
   <scm>
     <url>https://github.com/corona-warn-app/cwa-verification-server</url>
   </scm>
```